Dockerfile.builder

```diff
@@ -19,7 +19,14 @@ ENV GO_VERSION 1.13.6
 #
 # Build dependencies
 #
-RUN apt-get update \
+RUN dpkgArch="$(dpkg --print-architecture)"; \
+case "${dpkgArch##*-}" in \
+  arm64) libcdev="" ;; \
+  amd64) libcdev="libc6-dev-i386" ;; \
+  i386) libcdev="libc6-dev-i386" ;; \
+  *) libcdev="" ;; \
+esac; \
```

Comment:
Installing linc6-devi386 should no longer be needed after #10204 being merged. We no longer depend on headers from libc.